CMBLensing_spt3g_interface/CMBLensing_spt3g_interface.py

```diff
@@ -1,156 +0,0 @@
-### initialize PyJulia
-import sys
-import julia
-from .julia_jlcommand import jl
```

Comment:
Now that everything is in a single .jl file, what do we do about the `jl` command? The simplest thing would be to stick its contents in `CMBLensingSPT3GInterface.__init__()` maybe?

Reply:
I'd keep it its own separate file, but can certainly import it from that __init__.

Reply:
Aren't there path issues with importing from `__init__`? I was under the impression that `py"import .julia_jlcommand"` in the `__init__` of a Julia package wouldn't know to look in the package's `src` directory.

Reply:
I don't think I understood your question right the first time, but anyway I think the Python stuff is set up ok now. After following the install instructions in the readme I added, its just

```python
>>> from CMBLensingSPT3GInterface import jl

>>> jl("FlatMap(rand(2,2))") # will come out a FlatSkyMap in Python, etc...
```